doc/classes/String.xml

```diff
@@ -288,43 +327,43 @@
 			<return type="String" />
 			<param index="0" name="prefix" type="String" />
 			<description>
-				Returns a copy of the string with lines indented with [param prefix].
-				For example, the string can be indented with two tabs using [code]"\t\t"[/code], or four spaces using [code]"    "[/code]. The prefix can be any string so it can also be used to comment out strings with e.g. [code]"# "[/code]. See also [method dedent] to remove indentation.
-				[b]Note:[/b] Empty lines are kept empty.
+				Returns a copy of the string with lines indented with [param prefix]. Empty lines are kept empty. See also [method dedent] to remove indentation.
+				For example, the string can be indented with two tabs using [code]"\t\t"[/code], or four spaces using [code]"    "[/code]. The [param prefix] can be any [String], so it can also be used to comment out strings with e.g. [code]"# "[/code].
 			</description>
 		</method>
 		<method name="insert" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="position" type="int" />
 			<param index="1" name="what" type="String" />
 			<description>
-				Returns a copy of the string with the substring [param what] inserted at the given [param position].
+				Inserts [param what] at the given [param position] in the string.
 			</description>
 		</method>
 		<method name="is_absolute_path" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the string is a path to a file or directory and its starting point is explicitly defined. This includes [code]res://[/code], [code]user://[/code], [code]C:\[/code], [code]/[/code], etc.
+				Returns [code]true[/code] if the string is a path to a file or directory, and its starting point is explicitly defined. This method is the opposite of [method is_relative_path].
+				This includes all paths starting with [code]"res://"[/code], [code]"user://"[/code], [code]"C:\"[/code], [code]"/"[/code], etc.
 			</description>
 		</method>
 		<method name="is_empty" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the length of the string equals [code]0[/code].
+				Returns [code]true[/code] if the string's length is [code]0[/code] ([code]""[/code]). See also [method length].
 			</description>
 		</method>
 		<method name="is_relative_path" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the string is a path to a file or directory and its starting point is implicitly defined within the context it is being used. The starting point may refer to the current directory ([code]./[/code]), or the current [Node].
+				Returns [code]true[/code] if the string is a path to a file or directory, and its starting point is implicitly defined within the context it is being used. The starting point may refer to the current directory ([code]"./"[/code]), or the current [Node]. This method is the opposite of [method is_absolute_path].
```

Comment:
Yes it technically works with Nodes. No, I do not understand the Node remark either. It's just stupid and muddies the water, especially because I believe **NodePath** itself has the method to check for this. A note about it may still be necessary, however, because any normal human being uses a String parameter to get nodes.

Reply:
But then it's not a file or a directory path, like the sentence claims earlier

Reply:
Rewrote the mess entirely.